access.tex

```diff
@@ -0,0 +1,37 @@
+\section{Data Access Environment} \label{access}
+
+The Rubin data access environment provides data rights holders with access to all Rubin data products and services. 
+Prior to the start of Operations, all services for data access will be under active development and are provided on a shared-risk basis. 
+The Rubin data rights policy is described in  \citeds{RDO-013}.
+
+\subsection{Data Access Centers}
+
+Rubin data products will be served to the community from the US Data Access Center (US DAC) hosted on the Google Cloud Platform\footnote{\url{data.lsst.cloud}}.
+\hyperref[sec:dp01]{DP0.1} and \hyperref[sec:dp02]{DP0.2}  are already available via the RSP running at the US DAC.
```

Comment:
Could mention DP0.3 here?

Reply:
It has not been publicly released yet, and as I want to get this document out this week, I'd rather not.